prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <a href="/prep-html-links/index.html" alt="home page">Click here to go to the home page</a>
```

Comment:
Hi Daniel, this link should use a relative path back to the index file. `"../index.html"`.

Reply:
Hi Tim, the code has been updated thanks for the tip
<img width="596" alt="Screen Shot 2021-09-18 at 2 13 33 PM" src="https://user-images.githubusercontent.com/90487207/134075859-fa9a944a-adcd-457f-83ed-c9ff289bebee.png">
.

Reply:
Hi Tim , the code has been updated thank
<img width="644" alt="Screen Shot 2021-09-20 at 2 06 43 PM" src="https://user-images.githubusercontent.com/90487207/134075945-4e901095-15b0-42f4-9298-5b7fd3cd0d09.png">
s.